README.md

```diff
@@ -49,6 +49,7 @@ Completed:
 * [x] Provisioner: AWS EC2
 * [x] Provisioner: Azure
 * [x] Provisioner: Linode
+* [x] Provisioner: Hetzner
 * [x] `inletsctl delete` command
```

Comment:
Would be helpful to add example usage with Hetzner in README

Reply:
Added a create example on like 203-215 in the readme, right under Scaleway, also included region information to make it easy to choose for the users.